src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -196,8 +307,8 @@ public function setup(): void
     public function channel(): \AMQPChannel
     {
         if (null === $this->amqpChannel) {
```

Comment:
Can be checked via `\is_null()` for opcode optimisations :)

Reply:
Nope sorry, that's not our CS, and that wouldn't change anything actually :)